index.js

```diff
@@ -47,7 +49,8 @@ module.exports = () => {
 	});
 };
 
-module.exports.download = (win, url) => {
-	registerListener(win, {unregisterWhenDone: true});
+module.exports.download = (win, url, opts = {}) => {
+	opts.unregisterWhenDone = true;
```

Comment:
Use `Object.assign()` here. This will modify the user supplied object, and if the user uses the same object for multiple calls it will result in different behavior.